mpp/docs/Stubbed Projects.md

```diff
@@ -0,0 +1,36 @@
+# Stubbed Projects
+
+### List:
+- :annotation:annotation is stubbed by `:annotation:annotation-compatibility-stubs`, it depends on `androidx.annotation`
+- :collection:collection is stubbed by `:collection:collection-compatibility-stubs`, it depends on  `androidx.collection`
+
+### Purpose
+The need for "stubbed projects" appeared in this PR - https://github.com/JetBrains/compose-multiplatform-core/pull/1819
+
+YT ticket: https://youtrack.jetbrains.com/issue/CMP-7183/Infrastucture-No-need-to-publish-annotation-and-collection-libraries-anymore
```

Comment:
```suggestion
YT ticket: [CMP-7183: No need to publish annotation and collection libraries anymore](https://youtrack.jetbrains.com/issue/CMP-7183)
```

Also, it's private. If you want to mention it here, please open it to public

Reply:
fixed